README.md

```diff
@@ -1,54 +1,83 @@
 
-- **Cooperates with [TiDB](https://github.com/pingcap/tidb):** Thanks to the internal optimization, TiKV and TiDB can work together to be a compelling database solution with high horizontal scalability, externally-consistent transations, and support for RDBMS and NoSQL design patterns.
+- **Horizontal scalability**
 
+    With PD and carefully designed Raft groups, TiKV excels in horizontal scalability and can easily scale to 100+ TBs of data.
 
-## The TiKV Software Stack
+- **Consistent distributed transactions**
 
-![The TiKV software stack.](images/tikv_stack.png)
+    Similar to Google's Spanner, TiKV supports externally-consistent distributed transactions.
 
-- **Placement Driver:** Placement Driver (PD) is the cluster manager of TiKV. PD periodically checks replication constraints to balance load and data automatically.
-- **Store:** There is a RocksDB within each Store and it stores data into local disk.
+- **Coprocessor support**
+
+    Similar to Hbase, TiKV implements a coprocessor framework to support distributed computing.
+
+- **Cooperates with [TiDB](https://github.com/pingcap/tidb)**
+
+    Thanks to the internal optimization, TiKV and TiDB can work together to be a compelling database solution with high horizontal scalability, externally-consistent transactions, support for RDBMS, and NoSQL design patterns.
+
+## TiKV software stack
+
+![The TiKV software stack](images/tikv_stack.png)
+
+- **Placement Driver:** PD is the cluster manager of TiKV, which periodically checks replication constraints to balance load and data automatically.
+- **Store:** There is a RocksDB within each Store and it stores data into the local disk.
 - **Region:** Region is the basic unit of Key-Value data movement. Each Region is replicated to multiple Nodes. These multiple replicas form a Raft group.
 - **Node:** A physical node in the cluster. Within each node, there are one or more Stores. Within each Store, there are many Regions.
 
-When a node starts, the metadata of the Node, Store and Region are registered into PD. The status of each Region and Store is reported to PD regularly.
+When a node starts, the metadata of the Node, Store and Region are recorded into PD. The status of each Region and Store is reported to PD regularly.
 
+## Getting started
 
-## Your First Test Drive
+TiKV is a component of the [TiDB project](https://github.com/pingcap/tidb). To run TiKV you must build and run it with PD, which is used to manage the cluster. You can use TiKV together with TiDB. You can also use TiKV separately.
 
-We have a [Docker Compose](https://github.com/pingcap/tidb-docker-compose/) you can use to test out [TiKV](https://github.com/pingcap/tikv) and [TiDB](https://github.com/pingcap/tidb).
+We provide multiple deployment methods, but it is recommended to use the Ansible deployment for the production environment. The TiKV documentation is available on [TiKV's wiki page](https://github.com/pingcap/tikv/wiki/TiKV-Documentation).
```

Comment:
I'm not sure pointing people to production deployment guides is appropriate for a "Getting Started" section.